README.md

```diff
@@ -131,8 +124,8 @@ Path for Hyperkvasir dataset (must contain folders `labeled-images` and `segment
 Path for ERS dataset (must contain patient id directories e.g. `0001`)  
 - `--ers-use-seq`  
 Use sequences directory for ERS dataset (e.g. "seq_01"). Defaults to false.
-- `--ers-use-empty-masks`  
-Flag specifying whether frames with empty mask files that are labeled as not healthy will be used for segmentation. Empty masks files that are healthy will be used nonetheless. See [healthy flag section](#healthy-flag). Defaults to false. For training type `classification` it is overriden to true.
+- `--ers-use-empty-masks` 
+Flag specifying whether images with empty mask files should be used for segmentation. Independently, script will use empty mask files that belong to healthy classes. See [healthy flag section](#healthy-flag). Defaults to false. For training type `multilabel-classification` it is overridden to true.
```

Comment:
The trick is really that all empty mask files, if processed, are created as an fully white image. We should add this information here. I think we can also remove the healthy flag section

Reply:
Done